### ISSUE TITLE
日语无法处理长文本？

### ISSUE THREAD

**Issue by underdogliu** [FunAudioLLM/CosyVoice#152]
**Describe the bug**
当输入的文本过长的时候，模型无法合成整个句子，只有前半部分。不知道是否是我代码有问题？

**To Reproduce**
以下是相关的代码
```
from cosyvoice.cli.cosyvoice import CosyVoice
from cosyvoice.utils.file_utils import load_wav
import torchaudio

import os

os.system("export PYTHONPATH=third_party/Matcha-TTS")

cosyvoice = CosyVoice('pretrained_models/CosyVoice-300M')
# sft usage
print(cosyvoice.list_avaliable_spks())
output = cosyvoice.inference_sft("10年ぶりにあった友人は、ほっそりとしていて、生気が抜けてしまったように思えて仕方がなかった", '日语男')
torchaudio.save('sft.wav', output['tts_speech'], 22050)
```

**Expected behavior**
在有pretrained_models的前提下可直接运行。句子到“て”这个音之后就没有了。

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: No browser
 - Version: Latest one (cloned from Github)

**Comment by aluminumbox** [comment]
try change て、 to て. or split sentence manually

**Comment by underdogliu** [comment]
@aluminumbox 感谢回复！确实改变标点符号就能合成完整句子了。

不知道可否再问一个问题：请问日语的话目前在这个repo里面是只有一个说话人吗？请问相关服务还提供更多的日语或者多语种说话人吗？

**Comment by aluminumbox** [comment]
> @aluminumbox 感谢回复！确实改变标点符号就能合成完整句子了。
> 
> 不知道可否再问一个问题：请问日语的话目前在这个repo里面是只有一个说话人吗？请问相关服务还提供更多的日语或者多语种说话人吗？

yes we only provide 1 japanese sft speaker. the model only supports 5 language. you can finetune your own japanese speaker if you have data
